examples/complete/main.tf

```diff
@@ -8,6 +8,7 @@ module "vpc_prod" {
 
   cidr_block = "172.16.0.0/16"
 
+  enabled    = true
```

Comment:
```suggestion

```

No need to specify `enabled` here, it will be taken from the context after defining `var.enabled` similar to examples/complete/fixtures.enabled.tfvars

Reply:
thanks @aknysh. I did have that originally with the first stage of the test with the `-target` options always ensuring `enabled = true` in the context object.
However when the next apply runs with `fixtures.disabled.tfvars` specified, terraform suddenly finds the base vpc infrastructure needs to be destroyed and it also causes an error.

As an alternative to specifying `enabled = true` on each base infrastructure module, I could chain another context object and override the outer context enabled flag. Then pass the overridden label context to it.

```
module "vpc_infra_label" {
  source  = "cloudposse/label/null"
  version = "0.24.1"

  enabled = true

  context = module.this.context
}
```

Do you agree this will be a cleaner solution?

Reply:
Just a bump on this one please. Which is your preference?